### ISSUE TITLE
<project>_config.gpr : warning, no sources of language "Ada" in this project

### ISSUE THREAD

**Issue by pyjarrett** [alire-project/alire#783]
I received an unexpected warning due to an empty `<project>_config.gpr` file.

## Reproduction Steps

```bash
git checkout master # master is @ b45ea5d10f75dc0f39edde4ba198bede7711d131
gprbuild -j0 -P alr_env
cd ../
alire/bin/alr init --bin spacing_demo
cd spacing_demo
../alire/bin/alr build
```

## Short Error

```
spacing_demo_config.gpr:1:09: warning: there are no sources of language "Ada" in this project
```

## Full Output

```
 $ ../alire/bin/alr build
Setup
   [mkdir]        object directory for project Spacing_Demo
   [mkdir]        exec directory for project Spacing_Demo
spacing_demo_config.gpr:1:09: warning: there are no sources of language "Ada" in this project
Compile
   [Ada]          spacing_demo.adb
Bind
   [gprbind]      spacing_demo.bexch
   [Ada]          spacing_demo.ali
Link
   [link]         spacing_demo.adb
```

```
 $ cat config/spacing_demo_config.gpr
project Spacing_Demo_Config is
end Spacing_Demo_Config;
```

I'm not sure what Alire's plan is for this GPR file.

**Comment by Fabien-Chouteau** [comment]
> I'm not sure what Alire's plan is for this GPR file.

This is where the [crate configuration](https://github.com/alire-project/alire/blob/master/doc/catalog-format-spec.md#using-crate-configuration) and [auto gpr with](https://github.com/alire-project/alire/blob/master/doc/user-changes.md#automatic-gpr-with-now-in-crate-configuration) generate gpr code.

The project should be abstract and you won't see the error anymore. Fix here -> https://github.com/alire-project/alire/pull/785

**Comment by mosteo** [comment]
Fixed by #785
